app/controllers/api/v1/customers_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::CustomersController < ApplicationController
+  def index
+    render json: Customer.all.to_json(only: [:id, :first_name, :last_name])
```

Comment:
@s-espinosa This is the first way I found to limit the returned JSON. I think I'll change it to JBuilder to format it prettier.